### ISSUE TITLE
[Feature Request] Plase add option to make requests only if in card screen.

### ISSUE THREAD

**Issue by wemersonrv** [GijsGoudzwaard/nova-system-resources#21]
Hello, Package make requests even when it's in another screen. I think it's better if only make requests in screen the cards are in use.

In my case, i only use it in main dashboard, but when i go to any resource page, requests still happens.

**Comment by nibri10** [comment]
I have the same problem

**Comment by wemersonrv** [comment]
> I have the same problem

I think this is fine... **but**... It's make unnecessary requests and this can be improved!

**Comment by GijsGoudzwaard** [comment]
I was able to reproduce this issue. I'll fix this soon

**Comment by wemersonrv** [comment]
Hello. Any updates on that?

**Comment by GijsGoudzwaard** [comment]
I'm currently quite busy and I haven't found the time to fix this issue yet. A PR is welcome

**Comment by authanram** [comment]
https://github.com/GijsGoudzwaard/nova-system-resources/pull/25

Hope this helps.

**Comment by authanram** [comment]
@wemersonrv 

For now you can use my current master, till it's merged;

```
    "require": {
        ...
        "gijsg/system-resources": "dev-master",
        ...
    },
```

```
    "repositories": {
        ...
        "gijsg/system-resources": {
            "type": "vcs",
            "url": "https://github.com/authanram/nova-system-resources.git"
        }
    }
```

Good luck.

**Comment by wemersonrv** [comment]
> @wemersonrv
> 
> For now you can use my current master, till it's merged;
> 
> ```
>     "require": {
>         ...
>         "gijsg/system-resources": "dev-master",
>         ...
>     },
> ```
> 
> ```
>     "repositories": {
>         ...
>         "gijsg/system-resources": {
>             "type": "vcs",
>             "url": "https://github.com/authanram/nova-system-resources.git"
>         }
>     }
> ```
> 
> Good luck.

Gread job!

**Comment by GijsGoudzwaard** [comment]
This issue is fixed in #25. It can be used with version `1.0.4`.
